# Base configuration for the BumblebeeXB3 camera driver
BumblebeeXb3:
  ros__parameters:
    camera_model: "XB3"
    camera_name: "front_xb3"
    rectified_image_size_height: 384
    rectified_image_size_width: 512
    packet_multiplier: 0
    output_gray: false
    show_rectified_images: true
    show_raw_images: false


